packages/yarnpkg-shell/sources/index.ts

```diff
@@ -392,7 +392,10 @@ function makeCommandAction(args: Array<string>, opts: ShellOptions, state: Shell
   if (name === `command`) {
     return makeProcess(rest[0], rest.slice(1), opts, {
       cwd: npath.fromPortablePath(state.cwd),
-      env: state.environment,
+      env: {
+        ...state.environment,
+        PWD: npath.fromPortablePath(state.cwd),
```

Comment:
The berry shell is a shim for a BASH environment, and BASH's `PWD` is in the `/c/path/to/cwd` format so I think we should mimic that here.

Reply:
I disagree here. The shell is purely syntaxical - converting the environment variables is out of scope (otherwise we'd otherwise normalize the PATH etc, and we don't want to enter this rabbit hole).

Reply:
> The shell is purely syntaxical 

Fair enough, in that case the comment about unconditionally adding it in PowerShell / CMD applies.

> converting the environment variables is out of scope

There are two windows environments this can run in:

- PowerShell/CMD: no PWD is set
- BASH: PWD is a portable path

I'm interpreting "converting" differently than you, I think. I consider outputting a DOS-style path here a conversion.

In the end the point is purely one of preference. The following three commands all work in BASH: `ls /c/users`, `ls C:/Users`, `ls C:\\Users`, so I think whatever we output will work fine.